Windows.MSVC.toolchain.cmake

```diff
@@ -37,6 +37,7 @@
 # | CMAKE_VS_PLATFORM_TOOLSET_VERSION           | The version of the MSVC toolset to use. For example, 14.29.30133. Defaults to the highest available.                     |
 # | CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE | The architecture of the toolset to use. Defaults to 'x64'.                                                               |
 # | CMAKE_WINDOWS_KITS_10_DIR                   | The location of the root of the Windows Kits 10 directory.                                                               |
+# | VS_ENABLE_SPECTRE                           | Whether to compile/link with 'Spectre' mitigations enable. Defaults to 'ON' for non-Debug CMAKE_BUILD_TYPE builds.       |
```

Comment:
The language here is a bit confusing to me. The table defines variables. The default value for the `VS_ENABLE_SPECTRE` variable is not "ON". Spectre mitigations default to on when `VS_ENABLE_SPECTRE` is `""`.

Reply:
I switched to just controlling the use of the runtime with Spectre mitigations enabled - which seems to be more appropriate for the toolchain. Setting `VS_USE_SPECTRE_MITIGATION_RUNTIME` to be on/true will cause the toolchain to link with the runtime with Spectre mitigations. It defaults to OFF. This means that the mitigations (and associated performance affects) are off by default, and the expectation would be that if the runtime is being used to manipulate data that crosses a trust boundary, consumers would turn it ON.